src/components/Picker/IPickerSource.ts

```diff
@@ -0,0 +1,4 @@
+import { PeopleInfo } from './PeopleInfo';
```

Comment:
Move PeopleInfo out of components and into src/example.  The component is abstract and can have any item.  Let's make it generic:
```
export interface IPickerSource<T> {
     performFilter(filterString: string): Promise<T[]>,
}
```